### PR TITLE
Awesome-Books

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
- Use Es6 in the javascript file,

- Divide code into modules,

- Add index.js,

- Add index.html

- Add index.css

- Add package.json

- Add package-lock.json

### CODE REVIEW

**Review by Kingstalux — APPROVED** [review]
Hi @Godfrey-Alimony,
# Status: Approved ✔️ ✔️ ✔️ 
 
download

## Highlights 👏 
✅ App has the correct functionality 🔥 
✅ Good job using ES6 syntax and modules 👍 
✅ Linters are passing 👌 
✅ Good commit history 💯 

Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: components/AddNewBookSection.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  localStorage.setItem('formData', JSON.stringify(book));
+};
+
+const displaySuccess = () => {
+  const successMsg = document.getElementById('success-msg');
+  successMsg.textContent = 'Your book has been added!';
+  setTimeout(() => {
+    successMsg.textContent = '';
+  }, 3000);
+};
```
Comment (mrEckendonk) [inline components/AddNewBookSection.js:31]: Nicely done to give the user feedback that a book is added to the list. Great Idea to make the UI user-friendly.

**Review by mrEckendonk — COMMENTED** [review]
Hi @Godfrey-Alimony, 

We reviewed your code and there is nothing to say than Congratulations, well done and a great UI made. 🎉

**Comment by mrEckendonk** [comment]
Hi @Godfrey-Alimony.

Project approved. Tim to merge!

**Pull request merged**